addons/test_mail/tests/test_mail_activity.py

```diff
@@ -266,6 +266,27 @@ def test_action_feedback_attachment(self):
             self.assertEqual(attachment.res_id, activity_message.id)
             self.assertEqual(attachment.res_model, activity_message._name)
 
+    def test_chained_activities(self):
+        partner_model = self.env.ref('base.model_res_partner')
+        partner = self.env['res.partner'].create({
+            'name': 'Tester',
+        })
+        chained_activity = self.env['mail.activity'].create({
+            'summary': 'Test',
+            'activity_type_id': self.env.ref('test_mail.mail_act_test_chained_1').id,
+            'res_model_id': partner_model.id,
+            'res_id': partner.id,
+        })
+        chained_activity.action_feedback(feedback='Done')
+
+        self.assertFalse(chained_activity.exists())
+
+        next_activity = self.env['mail.activity'].search([('res_model_id','=', partner_model.id), ('res_id','=', partner.id)])
```

Comment:
We could also simply take ``next_activity = partner.activity_ids``, isn't it ? That way we use the interface field.

Reply:
ok done